external_resources/api.py

```diff
@@ -55,9 +58,52 @@ def is_external_url_broken(
     return is_url_broken(url)
 
 
-def is_backup_url_broken(
-    external_resource: WebsiteContent,
-) -> tuple[bool, Optional[int]]:
-    """Check if backup url of the provided WebsiteContent is broken"""
-    url = external_resource.metadata.get("backup_url", "")
-    return is_url_broken(url)
+def submit_url_to_wayback(
+    url: str,
+) -> Optional[str]:
+    """
+    Submit the external resource URL to the Wayback Machine and
+    return the response
+    """
+    if not url:
+        return None
+
+    headers = {
+        "Accept": "application/json",
+        "Authorization": (
+            f"LOW {settings.WAYBACK_MACHINE_ACCESS_KEY}:"
+            f"{settings.WAYBACK_MACHINE_SECRET_KEY}"
+        ),
+    }
+    params = {
+        "url": url,
+        "skip_first_archive": "1",
+    }
+
+    response = requests.post(WAYBACK_API_URL, headers=headers, data=params, timeout=30)
+    response.raise_for_status()
+    return response.json()
+
+
+def check_wayback_jobs_status_batch(job_ids: list[str]) -> list[dict]:
+    """
+    Check the status of multiple Wayback Machine jobs in batch.
+    """
+    if not job_ids:
+        return []
+
+    headers = {
```

Comment:
Since the headers are fixed and used multiple times, it would have been better to have them in a constants file.